--- embree-3.13.2/tutorials/external/catch.hpp~	2021-10-26 16:31:08.000000000 +0000
+++ embree-3.13.2/tutorials/external/catch.hpp	2021-12-23 22:25:49.948939136 +0000
@@ -8750,7 +8750,7 @@
 
     // 32kb for the alternate stack seems to be sufficient. However, this value
     // is experimentally determined, so that's not guaranteed.
-    constexpr static std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;
+    constexpr static std::size_t sigStackSize = 32768;
 
     static SignalDefs signalDefs[] = {
         { SIGINT,  "SIGINT - Terminal interrupt signal" },
